### ISSUE TITLE
Flagging for stealing source

### ISSUE THREAD

**Issue by arifszn** [0x1381/info#1]
This repo does not comply with the original [source](https://github.com/arifszn/ezprofile).
- This is not the fork of the original repo
- Your website impersonates that you are the sole creator of the work by showing `Made with Afshin and ❤️`. I made this repo by giving tons of my free time. You can't just modify some values and claim it to be yours.

Go through the [Please Read](https://github.com/arifszn/ezprofile#-please-read) section of the original source.

**Comment by 0x1381** [comment]
I have to do this Made with ezProfile ❤️?

**Comment by arifszn** [comment]
Just fork the repo. Then no need of `Made with ezProfile ❤️`. A star is also appreciated.

**Comment by 0x1381** [comment]
That is, the only problem is forking the repo?

**Comment by arifszn** [comment]
No, the only problem is you basically stole the repo and claimed it to be yours.

**Comment by 0x1381** [comment]
I wanted to solve this problem but I do not do it because you consider me a thief

**Comment by 0x1381** [comment]
You did not mention the copyright, you blame me

**Comment by 0x1381** [comment]
You can encrypt your code. Bye

**Comment by arifszn** [comment]
LOL. Ever heard of [DMCA takedown](https://www.dmca.com/FAQ/What-is-a-DMCA-Takedown)? Bye
